bin/create-phpdoc-index.php

```diff
@@ -1,170 +0,0 @@
-<?php
```

Comment:
This file seems to be unused, so I removed it.

Reply:
Delete my code, eh?!

Yes, it's obsolete.